paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass.cc

```diff
@@ -228,6 +228,49 @@ void CPUQuantizeSquashPass::FcDequantSquash(Graph* graph) const {
                   found_fc_dequant_squash_count);
 }
 
+void CPUQuantizeSquashPass::MultipleQuantizeSquash(Graph* graph) const {
+  GraphPatternDetector gpd;
+  patterns::MultipleQuantize multiple_quantize_pattern{gpd.mutable_pattern(),
+                                                       "multiple_quantize"};
+  multiple_quantize_pattern();
+
+  int found_multiple_quantize_squash_count = 0;
+  int removed_quantize = 0;
+  auto handler = [&](const GraphPatternDetector::subgraph_t& subgraph,
+                     Graph* g) {
+    VLOG(4) << "fuse multiple quantize ops";
+
+    GET_IR_NODE_FROM_SUBGRAPH(prev_op, prev_op, multiple_quantize_pattern);
+    GET_IR_NODE_FROM_SUBGRAPH(prev_out, prev_out, multiple_quantize_pattern);
+
+    auto* first_quant_op =
+        *(std::find_if(prev_out->outputs.begin(), prev_out->outputs.end(),
+                       [&](Node* node) { return node->Name() == "quantize"; }));
+    auto* first_quant_out = first_quant_op->outputs[0];
+    float scale = first_quant_op->Op()->GetAttrIfExists<float>("Scale");
```

Comment:
Please add a check if the `scale` is nonzero.

Reply:
Done